plugin/src/main/java/org/wildfly/swarm/plugin/maven/RepositorySystemSessionWrapper.java

```diff
@@ -175,4 +190,24 @@ public RepositoryCache getCache() {
     private RepositorySystemSession delegate;
 
     private DependencyGraphTransformer transformer;
+
+    private boolean excludeSwarm;
+
+    public class SwarmExcludedTraverser implements DependencyTraverser
+    {
+
+       @Override
+       public boolean traverseDependency(Dependency dependency)
+       {
+          return !SWARM_GROUP_ID.equals(dependency.getArtifact().getGroupId());
```

Comment:
Can we do this without playing with groupId string comparison?  I've tried hard in BuildTool/DependencyManager and the fraction-plugin to not assume any particular groupId for a fraction type of thing.